Gifski/VideoDropView.swift

```diff
@@ -14,6 +14,12 @@ class DropView<CompletionType>: SSView {
 		}
 	}
 
+	var dropView: NSView? {
+		didSet {
+			dropView?.isHidden = dropView == nil
```

Comment:
this is really interesting! but isn't it effectively the same as `dropView?.isHidden = false`?

Reply:
You're probably right. I'm just gonna drop it as we don't really need that handling at all. We only set `dropView` once. I did it for the label as back then I thought it was smart to make the drop view as reusable as possible, but that was honestly a mistake. It's just made it harder to customize it.